### DISCUSSION TITLE
What is being done about incoming issues?

### DISCUSSION THREAD

**Discussion by ffMathy** [pulumi/pulumi#9639] [General]
<img width="951" alt="image" src="https://user-images.githubusercontent.com/2824010/169525654-2aa974cb-432f-4d36-989a-dc35ace76708.png">

See attached screenshot. It seems as if the amount of new issues being opened is constantly higher than the amount of issues being closed. What is being done about this problem?

I love the fast response of the team and Pulumi in general, but when I see something like this, it makes me a bit worried.

**Comment by stack72** [comment]
Hi!

This is something that is really important to us internally and we actually have internal OKRs around reducing this number. One thing I will say is that we are still going through a lot of growth in Pulumi engineering. This means we have a plan to get these numbers down :)

I hope this helps?

Paul

**Reply by ffMathy** [reply]
That really helps. Thank you! 🙏
